custom_components/better_thermostat/helpers.py

```diff
@@ -255,7 +256,7 @@ async def startup(self):
 			self._hvac_mode = HVAC_MODE_OFF
 		self._bt_hvac_mode = self._hvac_mode
 		self._last_window_state = self.window_open
-		self._cur_temp = float(sensor_state.state)
+		self._cur_temp = convert_to_float(sensor_state.state)
```

Comment:
Well, you need three arguments for convert_to_float: value, instance_name, and context.

Reply:
@KartoffelToby I fixed it in https://github.com/KartoffelToby/better_thermostat/pull/360/commits/f5a1599e807c7dd79f9617a2ed610f6d945d02c4